client/app/kubernetes/service/reset/kubernetes-service-reset.html

```diff
@@ -28,5 +28,18 @@
                 </oui-radio>
             </oui-radio-group>
         </oui-field>
+        <oui-field data-label="{{ ::'kube_service_reset_version' | translate }}">
+            <label class="oui-select">
+                <select class="oui-select__input"
+                        id="resetVersion" name="resetVersion"
+                        data-ng-model="ctrl.model.version"
+                        data-ng-options="version for version in ctrl.availableVersions">
+                    <option value="">&nbsp;</option>
```

Comment:
Is this `<option>` is needed?

Reply:
Yes because we want version non mandatory. So by default it should be null/empty value

Reply:
Wouldn't something like `No version` be more explicit ?

Reply:
It has been asked to let it blank... :-$